src/main/services/window-manager.ts

```diff
@@ -77,33 +86,66 @@ export class WindowManager {
   public static createSystemTray(language: string) {
     const tray = new Tray(trayIcon);
 
-    const contextMenu = Menu.buildFromTemplate([
-      {
-        label: t("open", {
-          ns: "system_tray",
-          lng: language,
-        }),
-        type: "normal",
-        click: () => {
-          if (this.mainWindow) {
-            this.mainWindow.show();
-          } else {
-            this.createMainWindow();
-          }
+    const updateSystemTray = async () => {
+      const games = await gameRepository.find({
+        where: {
+          isDeleted: false,
+          executablePath: Not(IsNull()),
         },
-      },
-      {
-        label: t("quit", {
-          ns: "system_tray",
-          lng: language,
-        }),
-        type: "normal",
-        click: () => app.quit(),
-      },
-    ]);
+        take: 5,
+        order: {
+          lastTimePlayed: "DESC",
+        },
+      });
+
+      const recentlyPlayedGames: Array<MenuItemConstructorOptions | MenuItem> =
+        games.map(({ title, id, executablePath }) => ({
+          label: title,
+          type: "normal",
+          click: async () => {
+            if (!executablePath) return;
+            await gameRepository.update({ id }, { executablePath });
```

Comment:
just a question, why is it needed to update the executablePath on click?

Reply:
The same thing happens here as in the sidebar when we select a game in the library.
Without the update, we will not know which game the user launched last time and the game will not be first in the list.

Reply:
Ok, code looks fine, I will run and test it today. Really cool feature, thanks!

Reply:
This update was still in my mind so I investigated and this update is not needed right here. The process-watcher is the responsible to update the lastTimePlayed. 

We update the executablePath in the openGame function because there may be a case where the openGame receives the executablePath as an argument.

For your feature, this isn't really needed. I just tested and your feature works correctly removing this call. Could you just remove it? Then all should be fine to merge!

Reply:
Yes, you are right, I removed update, but in addition I discovered that lastTimePlayed is a utc string and it was not sorted correctly, I had to sort by updateAt, but I don’t know if it’s correct to sort by it.

Reply:
Yes, I think your idea is good for now. We will probably work on a migration and update so the DB saves the lastTimePlayed as timestamp. This will solve some issues that will happen sorting by updateAt too